src/pages/VideoUploadBox.tsx

```diff
@@ -0,0 +1,126 @@
+import { useTranslation } from 'react-i18next';
+
+import { Button, ButtonVariants } from '@/ui/Button';
+import { Icons } from '@/ui/Icon';
+import { Image } from '@/ui/Image';
+import { Inline } from '@/ui/Inline';
+import type { StackProps } from '@/ui/Stack';
+import { getStackProps, Stack } from '@/ui/Stack';
+import { Text, TextVariants } from '@/ui/Text';
+import { getValueFromTheme } from '@/ui/theme';
+import { Title } from '@/ui/Title';
+
+const getValue = getValueFromTheme('videoUploadBox');
+
+type ImageType = {
+  parsedId: string;
+  description: string;
+  copyrightHolder: string;
+  thumbnailUrl: string;
+  isMain: boolean;
+  file: File;
+};
+
+type Video = {
+  id: string;
+  url: string;
+  embedUrl: string;
+  language: string;
+  copyrightHolder: string;
+};
+
+type VideoEnriched = Video & {
+  thumbnailUrl: string;
+};
+
+type Props = StackProps & {
+  videos: VideoEnriched[];
+  onClickAddVideo: () => void;
+  onClickDeleteVideo: (videoId: string) => void;
+};
+
+const VideoUploadBox = ({
+  videos,
+  onClickAddVideo,
+  onClickDeleteVideo,
+  ...props
+}: Props) => {
+  const { t } = useTranslation();
+
+  return (
+    <Stack spacing={2}>
+      <Title size={3}>{t('videos.title')}</Title>
+      <Stack
+        flex={1}
+        spacing={4}
+        padding={4}
+        backgroundColor={getValue('backgroundColor')}
+        justifyContent="center"
+        css={`
+          border: 1px solid ${getValue('borderColor')};
+        `}
+        {...getStackProps(props)}
+      >
+        <Stack
+          spacing={2}
+          maxHeight={380}
+          css={`
+            overflow: auto;
+          `}
+        >
+          {videos.map(({ id, url, thumbnailUrl }, index, imagesArr) => {
+            const thumbnailSize = 80;
```

Comment:
Same here, I think you can define this as  a global `const`?

Reply:
Yep nice catch